app/views/admin_request/hidden_user_explanation.html.erb

```diff
@@ -1,4 +1,4 @@
-<%= _("Dear {{name}},", :name => name_to) %>
+<%= _("Dear {{name}},", :name => name_to.html_safe) %>
```

Comment:
We're going to try making this a text template rather than html.

Reply:
Waiting to see if the changes I've made so far break anything I'd not thought about. Then I think the best place to test the textarea/email to user to say their request has been hidden as vexatious content is the `ServicesController` spec